src/test/FuzzerImpl.cpp

```diff
@@ -1214,6 +1142,31 @@ TransactionFuzzer::initialize()
         ltx.commit();
     }
 
+    // Get the list of ledger entries created during setup to place into
+    // mStoredLedgerKeys.
+    std::vector<LedgerEntry> init, live;
+    std::vector<LedgerKey> dead;
+    ltxOuter.getAllEntries(init, live, dead);
+
+    // Setup should only create entries; there should be no dead entries, and
+    // only one "live" (modified) one:  the root account.
+    assert(dead.empty());
+    assert(live.size() == 1);
+    assert(live[0].data.type() == ACCOUNT);
+    assert(live[0].data.account().accountID == root.getPublicKey());
+
+    // If we ever create more ledger entries during setup than we have room for
+    // in mStoredLedgerEntries, then we will have to do something further,
+    // perhaps such as breaking that array up into one array per ledger entry
+    // type (or one array for each of some ledger entry types and a direct
+    // byte<->entry correspondence for those entry types which admit one).
+    assert(init.size() <= FuzzUtils::NUM_VALIDATED_LEDGER_KEYS);
+
+    // Store the ledger entries created during setup in mStoredLedgerKeys.
+    std::transform(
+        init.cbegin(), init.cend(), mStoredLedgerKeys.begin(),
+        [](auto const& initEntry) { return LedgerEntryKey(initEntry); });
```

Comment:
Nitpick: you should be able to just pass `LedgerEntryKey` here, no lambda required.

Reply:
Done.